src/shared/constants/balance-questions.cat-a-mod2.constants.ts

```diff
@@ -1,23 +1,22 @@
 import { VehicleChecksQuestion } from '../../providers/question/vehicle-checks-question.model';
 
-// TODO - PREP-AMOD2: Add balance questions (MES-4670)
 export default [
   {
-    code: 'T01',
+    code: 'Q01',
```

Comment:
There have been discussions on other categories where we shouldn't have leading zeros in the questions so it should be Q1, Q2 ... Can you double check if we should or shouldn't here

Reply:
Who is the best person to make this decision?